modules/nlu/src/views/full/intents/utterances-state-utils.ts

```diff
@@ -1,11 +1,22 @@
+import { parseUtterance } from 'common/utterance-parser'
```

Comment:
**pointing the attention of reviwers on this line**

Reply:
What is special there ? Common is meant to be accessible everywhere, I don't see any issue here

Reply:
just to point out true changes inside this refactor